src/styles/03-organisms/accordion/_accordion.scss

```diff
@@ -3,11 +3,11 @@
   border: 1px solid $gray-light;
   color: inherit;
 
-  > button {
+  &__button {
     border: unset;
   }
 }
 
 .accordion-content {
-  @include space-inset-xs;
+  padding: $space-xs 0;
```

Comment:
This change looks like a regression. Now items within the accordion butt right up against the sides: 

![Screen Shot 2020-04-20 at 10 56 44 AM](https://user-images.githubusercontent.com/1720093/79766179-a11f4180-82f5-11ea-8e32-70af78c19b19.png)

Reply:
hmm so we want it on three sides, then.  But it doesBecause an inset on all of the box is going to cause the scrollbar to float away from the side like this - 
![Screen Shot 2020-04-20 at 11 45 18 AM](https://user-images.githubusercontent.com/3579472/79771189-708ed600-82fc-11ea-8d47-8545fd39bc66.png)

It seems like this should be in the `unordered-list` style instead of this, but how to make it work?

Reply:
`padding: $space 0 $space $space`

Reply:
OK!  Still feels weird to have it in `accordion` rather than `unordered-list` but I guess we can deal with that in the future ...

Reply:
I think it makes sense here still—let's make sure we both are understanding this correctly later